### ISSUE TITLE
[Idea] stopping cs

### ISSUE THREAD

**Issue by farfromsubtlee** [MCCTeam/Minecraft-Console-Client#2045]
### Console Client Version

v1.17.1

### Describe your problem

Is there a way to stop the cs file running forever? I'm using OreMiner.cs but I can't stop it without closing the client. Is there a solution to this? For example /stop OreMiner.cs

Can you help me to stop the client without closing it?

**Comment by farfromsubtlee** [comment]
After a few hours of effort, I solved my problem.

Inspired by SugarCaneFarmer.cs and added "bool miner=".
I also added custom command. Initially it returns false. Then, when /miner start is typed, it returns true and mining starts. When I type /miner stop, it stops.
Maybe he needs it. Or u can integrate it directly into OreMiner.cs. I think it's a useful feature.
Here it is;
```c#
//MCCScript 1.0

MCC.LoadBot(new OreMiner());

//MCCScript Extensions

/// <summary>
/// This bot can mine blocks that auto-spawn at given locations
/// </summary>
public class OreMiner: ChatBot
{
    private bool miner = false;
    // === CONFIG - REPLACE BLOCK LOCATION x y z VALUES HERE ===
    List<Location> location = new List<Location>()
    {
        new Location(x, y, z),
        new Location(x2, y2, z2),
        new Location(x3, y3, z3),
        // Add more here
    };
    // === END OF CONFIG ===
    int index = 0;

    public override void Initialize()
    {
        LogToConsole("Bot enabled, please use /miner to start it");
	RegisterChatBotCommand("miner", "Automatic mining", "/miner start/stop", CommandHandler);
    }

    public override void Update()
    {
	if(miner == true)
	{
	    Material blockType = GetWorld().GetBlock(location[index]).Type;
	    switch (blockType)
	    {
		    //Adjust here block types to mine
		    case Material.Cobblestone:
		    case Material.Stone:
		    case Material.DiamondOre:
		    case Material.EmeraldOre:
		    case Material.GoldOre:
		    case Material.IronOre:
		    case Material.CoalOre:
		    case Material.LapisOre:
		    case Material.RedstoneOre:
		    case Material.NetherQuartzOre:
			    DigBlock(location[index]);
			    break;
	    }
	    index++;
	    if (index >= location.Count)
            index = 0;
	}
    }
    public string CommandHandler(string cmd, string[] args)
    {
        if (args.Length > 0)
        {
            switch (args[0])
            {
                case "stop":
		    miner = false;
		    return "Stop mining...";
	        case "start":
	    	    miner = true;
		    return "Start mining...";
            }
        }
        return "Usage: /miner start/stop";
    }
}

**Comment by milutinke** [comment]
Implemented in #2167
Will be merged soon.
